### PR TITLE
Tweaks to get end to end deployment working

### PR DESCRIPTION
**Pull request by jackgr**
There's still an issue in expandybird (#485) but this PR fixes several blockers.

/cc @sparkprime

### CODE REVIEW

**Comment by sparkprime** [comment]
Expandybird tests need updating to have the right type string

File: cmd/manager/chartrepos.go [hunk 1]
```diff
@@ -58,6 +58,14 @@ func addChartRepoHandlerFunc(w http.ResponseWriter, r *http.Request, c *router.C
 		return nil
 	}
 
+	if string(cr.Format) == "" {
+		cr.Type = repo.GCSRepoType
```
Comment (sparkprime) [inline cmd/manager/chartrepos.go]: I dunno about the context of this code but this looks suspiciously like a copy/paste error from below

**Comment by jackgr** [comment]
Fixed chart naming.

File: cmd/manager/chartrepos.go (hunk 1 above)
Comment (jackgr) [inline cmd/manager/chartrepos.go]: Good catch. Fixed.

**Comment by sparkprime** [comment]
LGTM

**Comment by jackgr** [comment]
Thanks, @sparkprime!

**Comment by jackgr** [comment]
#474

**Pull request merged**